tests/run-make/multiple-emits/rmake.rs

```diff
@@ -0,0 +1,13 @@
+use run_make_support::{cwd, rustc};
+
+fn main() {
+    rustc().input("foo.rs").emit("asm,llvm-ir").output(cwd().join("out")).run();
```

Comment:
`cwd()` is the actual working directory, so it's enough to just use `.output("out")`, to make this simpler.